local-cli/runIOS/runIOS.js

```diff
@@ -82,20 +81,44 @@ function _runIOS(argv, config, resolve, reject) {
     '-derivedDataPath', 'build',
   ];
   console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
-  child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'inherit'});
 
-  const appPath = `build/Build/Products/Debug-iphonesimulator/${inferredSchemeName}.app`;
-  console.log(`Installing ${appPath}`);
-  child_process.spawnSync('xcrun', ['simctl', 'install', 'booted', appPath], {stdio: 'inherit'});
+  let appPath = `build/Build/Products/Debug-iphonesimulator/${inferredSchemeName}.app`;
+  const xcodeBuildProcess = child_process.spawn('xcodebuild', xcodebuildArgs);
+
+  xcodeBuildProcess.stdout.on('data', (data) => {
+    console.log(data.toString().replace(/\s+$/g, ''));
 
-  const bundleID = child_process.execFileSync(
-    '/usr/libexec/PlistBuddy',
-    ['-c', 'Print:CFBundleIdentifier', path.join(appPath, 'Info.plist')],
-    {encoding: 'utf8'}
-  ).trim();
+    // search this part of the process output for a path to the generated app and replace default
+    const appPathFromLog = data.toString().match(/Touch (build\/Build\/Products\/.*\/.*\.app)/);
+    if (appPathFromLog) {
+      appPath = appPathFromLog[1];
+    }
+  });
 
-  console.log(`Launching ${bundleID}`);
-  child_process.spawnSync('xcrun', ['simctl', 'launch', 'booted', bundleID], {stdio: 'inherit'});
+  xcodeBuildProcess.stderr.on('data', (data) => {
```

Comment:
Pass `[process.stdin, 'pipe', process.stderr]` as the stdio option and you don't need this at all.